src/models/projectModels.ts

```diff
@@ -2,6 +2,13 @@ export const PROJECT_TITLE_NONE = "None";
 export const PROJECT_TITLE_CREATE_NEW = "Create new...";
 
 export type Period = "daily" | "weekly" | "monthly" | "yearly";
+export const periodOptions = [
+  "daily" as Period,
```

Comment:
I don’t think you need to typecast all of these? I think it will work if you just add a type annotation:
```typescript
export const periodOptions: Period[]
```

Reply:
oh cool